app/assets/javascripts/attractions.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+  $('#add-attraction').click(function(event){
+    event.preventDefault();
+
+    var url = $(this).attr('href');
+
+    $.ajax({
+      url: url,
+      type: "get",
+      beforeSend: function(xhr) { xhr.setRequestHeader('X-CSRF-Token', $('meta[name=csrf-token]').attr('content'))},
+      dataType: 'json'
+    }).done(function(data){
+        loadMarkers(data)
+    });
```

Comment:
You should always handle `error` in JS as well, because from the user's perspective nothing is happening.